lua/lazyvim/plugins/extras/coding/avante.lua

```diff
@@ -0,0 +1,34 @@
+return {
+  {
+    "yetone/avante.nvim",
+    init = function()
+      require("avante_lib").load()
+    end,
+    event = "VeryLazy",
+    -- we want to use head for now, since the releases are not frequent
+    version = false,
```

Comment:
This is already default in [LazyVim](https://github.com/LazyVim/starter/blob/cb6349c8ae922d1c5745574f4d17b44f2731d451/lua/config/lazy.lua#L30), so it's not really needed?